app/models/concerns/changed_at.rb

```diff
@@ -0,0 +1,11 @@
+module ChangedAt
+  extend ActiveSupport::Concern
+
+  class_methods do
+  private
+
+    def timestamp_attributes_for_update
+      super + %w[changed_at]
```

Comment:
I think this could do with a code comment explaining what effect this has and why we needed to do it